spec/apinew.json

```diff
@@ -0,0 +1,7448 @@
+{
```

Comment:
What is this file and the other apiold.json for ?

Reply:
oops deleted :)